### PR TITLE
tidb-binlog: add decription for initial-commit-ts

### PR DESCRIPTION
**Pull request by Jingyuma**
The doc doesn't give description to the value '0' before v3.0.6, this may bring confuse between '0' and '-1'.
I'm not sure if this should be add, please help to review if the description should be add.

### What is changed, added or deleted? (Required)

Add description of the 0 value of initial-commit-ts.

### Which TiDB version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: tidb-binlog/tidb-binlog-configuration-file.md [hunk 1]
```diff
@@ -173,7 +173,7 @@ aliases: ['/docs-cn/v2.1/tidb-binlog/tidb-binlog-configuration-file/','/docs-cn/
 ### initial-commit-ts
 
 * 指定从哪个 commit timestamp 之后开始同步。这个配置仅适用于初次开始同步的 Drainer 节点。如果下游已经有 checkpoint 存在，则会根据 checkpoint 里记录的时间进行同步。
-* 默认：`-1`。Drainer 会从 PD 得到一个最新的 timestamp 作为初始时间。
+* 默认：`-1`。Drainer 会从 PD 得到一个最新的 timestamp 作为初始时间。(v3.0.6之前，默认：`0`，即不对初始时间做限制。)
```
Comment (july2993) [inline tidb-binlog/tidb-binlog-configuration-file.md:176]: how about "会从每个 pump 最老的数据开始拉取"

**Comment by Jingyuma** [comment]
HI， what should I do with this PR now?

File: tidb-binlog/tidb-binlog-configuration-file.md (hunk 1 above)
Comment (july2993) [inline tidb-binlog/tidb-binlog-configuration-file.md:176]:
```suggestion
* 默认：`0`。Drainer 会从每个 pump 最老的时间点开始拉取数据。
```

**Review by july2993 — APPROVED** [review]

**Comment by july2993** [comment]
> HI， what should I do with this PR now?

2.1 默认值应该是 0，文档分大版本的，所以这里应该改下默认值

建议按我 Suggested change 改下？

**Comment by Jingyuma** [comment]
> > HI， what should I do with this PR now?
> 
> 2.1 默认值应该是 0，文档分大版本的，所以这里应该改下默认值
> 
> 建议按我 Suggested change 改下？

已经修改。

**Review by july2993 — APPROVED** [review]

**Review by yikeke — APPROVED** [review]
LGTM

**Pull request merged**